app/services/PropertyService.scala

```diff
@@ -662,4 +663,18 @@ class PropertyService @Inject() (
       res <- persistAnswers(ctx.toJourneyContext(JourneyName.RentARoomAllowances), rentARoomAllowancesStoreAnswers)
     } yield res
   }
+
+  def saveForeignPropertiesInformation(
+    ctx: JourneyContext,
+    nino: Nino,
+    foreignPropertiesInformation: ForeignPropertiesInformation
+  )(implicit hc: HeaderCarrier): EitherT[Future, ServiceError, Boolean] =
```

Comment:
I think we don't need of EitherT here.

Reply:
We don't need a for comprehension but the persistAnswers type is EitherT